src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassical.cpp

```diff
@@ -0,0 +1,266 @@
+/**
+ * @file GenClassical.cpp
+ * @author Abdourahman Barry (abdourahman@vt.edu)
+ * @brief Definition of a Classical generator model.
+ *
+ *
+ */
+
+#include "GenClassical.hpp"
+
+#include <cmath>
+#include <iostream>
+
+#include <Model/PhasorDynamics/Bus/Bus.hpp>
+
+#define _USE_MATH_DEFINES
+
+namespace GridKit
+{
+  namespace PhasorDynamics
+  {
+    /*!
+     * @brief Constructor for a pi-model branch
+     *
+     * Arguments passed to ModelEvaluatorImpl:
+     * - Number of equations = 0
+     * - Number of independent variables = 0
+     * - Number of quadratures = 0
+     * - Number of optimization parameters = 0
+     */
+    template <class ScalarT, typename IdxT>
+    GenClassical<ScalarT, IdxT>::GenClassical(bus_type* bus, int unit_id)
+      : bus_(bus),
+        busID_(0),
+        unit_id_(unit_id),
+        p0_(0),
+        q0_(0),
+        H_(3.),
+        D_(0.),
+        Ra_(0.),
+        Xdp_(.5)

```

Comment:
```suggestion
        H_(3.0),
        D_(0.0),
        Ra_(0.0),
        Xdp_(0.5)

```
Adding this for consistency and readability. Floating point numbers should have at least 1 digit before and 1 digit after the decimal.